tests/tests/test_rtk_sync.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+from pathlib import Path
+
+FRONT_FOLDER = Path(__file__).parent.parent.parent / "front"
```

Comment:
```suggestion
FRONT_FOLDER = Path(__file__).parents[2] / "front"
```

But it's not too important, you can merge as it is